python/tvm/micro/contrib/zephyr.py

```diff
@@ -40,6 +38,7 @@
 from ..transport import debug
 from ..transport import file_descriptor
 
+from ..transport import serial
```

Comment:
why are changes in this file necessary with this PR , can this be split out into it's own Zephyr specific PR ?

Reply:
Zephyr support was previously merged in #6603. this file adds a serial-specific transport, which is useful only when running Zephyr against real hardware. the VM "enables" this in TVM by providing a standardized developer environment. it's still possible to use this outside the VM, too, but I think for the purposes of reproducing bugs, we will prefer to use the VM as a reference where possible.